main.cpp

```diff
@@ -4,7 +4,20 @@ using namespace std;
 
 int main()
 {
-  cout << "Hey Mel!";
+  int size;
+  char character;
+  cout << "Enter a size for your triangle";
+  cin  >>  size;
+  cout << "Enter a character: ";
+  cin  >>  character;
```

Comment:
I still have not figured this out. My brain is tired. Had some meds and can't keep my eyes open so I'll research when I'm fresh. I'm going to add what I've got so far. Let's see what else you have up your sleeve...